buildroot/package/recovery/recovery.mk

```diff
@@ -33,6 +33,7 @@ define RECOVERY_INSTALL_TARGET_CMDS
 	$(INSTALL) -m 0755 package/recovery/unicode-fonts/NanumBarunGothicBold.ttf $(TARGET_DIR)/usr/lib/fonts/NanumBarunGothicBold.ttf
 	$(INSTALL) -m 0755 package/recovery/data/data $(TARGET_DIR)/usr/data
 	$(INSTALL) -m 0644 $(@D)/cmdline.txt $(BINARIES_DIR)/cmdline.txt
+
```

Comment:
Could remove this change?

Reply:
yeah, small remnant from separating the profile PR.